lib/services/http_service.rb

```diff
@@ -42,6 +53,9 @@ def make_request(type = :get, endpoint = '/', query_params = nil, body = nil)
 
       http = Net::HTTP.new(uri.host, uri.port)
 
+      # http.set_debug_output $stderr
```

Comment:
as a practice, we generally dont leave commented out code in the code + blank lines like this.